templates/blog/product-cards/_desktop_ubuntu-desktop-for-organisations.html

```diff
@@ -0,0 +1,18 @@
+<div class="p-card js-product-card u-hide">
+    <img class="p-card__thumbnail"
+         src="https://assets.ubuntu.com/v1/9e27d14a-Ubuntu%20logo@2x.png"
+         alt="ubuntu logo" />
```

Comment:
We do not add "logo" word to mention it's logo. Just leaving out the name will be good enough

Reply:
Please refer [here](https://docs.google.com/document/d/12nLPp3ktElJCNl_i3q5fGOjWMPhPXXFsIJGH-AuxduQ/edit) for all alt="" related info